services/ResumeService.ts

```diff
@@ -61,4 +61,15 @@ export default class ResumeService {
       .resume(txn)
       .findByUserUuid(user.uuid));
   }
+
+  public async deleteResume(uuid: string, user: UserModel): Promise<ResumeModel> {
+    return this.transactions.readWrite(async (txn) => {
+      const resumeRepository = Repositories.resume(txn);
+      const resume = await resumeRepository.findByUuid(uuid);
+      if (!resume) throw new NotFoundError('Cannot find a resume of given uuid');
```

Comment:
nit: to match other "not found" error messages in other services
```suggestion
      const resume = await resumeRepository.findByUuid(uuid);
      if (!resume) throw new NotFoundError('Resume not found');
```